roles-and-groups.md

```diff
@@ -0,0 +1,153 @@
+- Keeps making bad decisions (their contributions are negatively affecting the project's performance).
+
+More on maintainer guidelines can be found on the [**LFDT TAC site**](https://lf-decentralized-trust.github.io/governance/governing-documents/MAINTAINERS-file.html).
+
+### Voting
+
+When it comes to electing and voting for new committers or new maintainers, the project should consider the following:
+
+For candidates seeking promotion to Committers or Maintainers:
+- The candidate needs to show participation in the project by contributing with code, issues, discussions, reviews, etc.
+- These contributions need to show proper understanding of the project and quality of the work.
+- 3 to 6 months worth of contributions is considered a good minimum for candidates to be selected for a role promotion.
+- A Committer or Maintainer will be considered inactive if they shown no activity for 6 months at least, in such case, they could be selected for removal.
+- A Committer can maintain their project status independently from their employment.
+
+For Maintainers and Committers voting on new candidates:
+- When a candidate is selected for becoming a Committer or a Maintainer, this proposal needs to be made via PR request against the config.yaml in the governance repo.
+- An example of a promotion can be found in the following [PR](https://github.com/hiero-ledger/governance/pull/176).
+- Once the PR is created, all the maintainers of the repo in question need to be added as reviewers and their vote should be provided.
+- The creator of the PR should notify all maintainers that a vote is now open.
+- Maintainers can vote in favor of the candidate's promotion by approving the PR with a comment indicating approval.
+- Maintainers can vote against the candidate's promotion by posting a comment in the PR along with their explanation.
+- Maintainers can abstain from voting by posting a comment in the PR along with their explanation.
+- The comment associated with a maintainers vote should be formatted as `[APPROVE|REJECT|ABSTAIN] - <explanation>`.
+- Maintainers are required to vote, if a maintainer does not participate in five consecutive votes they will be considered "inactive" which can be grounds for removal of maintainer status.
+- Once the PR has gathered the majority of the votes, the PR can be merged and the candidate will be officially part of their new Committer or Maintainer teams.
+- If a PR remains inactive for 2 days, it is a responsibility of the PR owner to reach out to the reviewers via Discord, bring it up in Community calls or reach out for help in the [hiero-general Discord channel](https://discord.com/channels/905194001349627914/1289954446712770600).
+- If a PR shows no activity for 2 weeks, it will be considered stale and it will be closed with no promotion.
+- A Maintainer or Committer can also nominate a candidate for removal from the teams if the candidate has not shown any activity for 6 months at least or is not responsive.
```

Comment:
Another instance of wording here that slightly conflicts with previous wording; I think I understand, really it's just a matter of consistency to help avoid confusion. E.g.:

> Project maintainers or the TSC may remove a maintainer

Whereas here, the doc only says that Maintainers/Committer 'can also nominate a candidate for removal...'

Or maybe I read it wrong–can Committers/Maintainers remove other Committers/Maintainers on legitimate grounds without a PR to change the config?

Reply:
yes.. both committers and maintainers can make a nomination to remove an inactive candidate in an effort to "clean up" their project and evaluate if they need to promote someone else. I will fix this wording in my next post. In general i think the words "also" is causing issues, my apologies..